### PR TITLE
Feat/refactor

### PR DESCRIPTION
**Pull request by giangbui**
- [x] Describe what this pull request does.
- [x] Make sure all tests pass.
- [x] Maintain or increase test coverage.
- [x] Black the code.
- [x] Test manually.
- [x] Remove empty sections below.

### New Features
- support renaming enum

### Improvements
- refactor the code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,15 +115,30 @@ Python SDK to create, explore and modify PFB files.
...
+### Rename enum
+
+    usage: pypfb rename enum [-h] -i INPUT -o OUTPUT --field_name FILED_NAME --val_from VALUE_FROM
+                                   --val_to VALUE_TO
+    
+    optional arguments:
+      -h, --help            show this help message and exit
+      -i INPUT, --input INPUT
+      -o OUTPUT, --output OUTPUT
+      -- field_name FIELD_NAME
```
Comment (m0nhawk) [inline README.md]: Extra space for `field_name`.

File: README.md [hunk 2]
```diff
@@ -115,15 +115,30 @@ Python SDK to create, explore and modify PFB files.
...
+      --val_from VALUE_FROM
+      --val_to VALUE_TO
+      
 
 ## Examples
```
Comment (m0nhawk) [inline README.md]: The whole section with examples should refer to PFB tool as `pfb`, which is added on package installation. Users shouldn't refer to `pypfb/__main__.py`.

File: pypfb/__main__.py [hunk 3]
```diff
@@ -139,20 +132,30 @@ def main():
...
-            rename_node(args.input, args.output, args.name_from, args.name_to)
+            Gen3PFB(args.input).rename_node(args.output, args.name_from, args.name_to)
+        elif args.rename == "enum":
+            Gen3PFB(args.input).rename_field_enum(args.output, args.field_name, args.val_from, args.val_to)
+
 
 
 if __name__ == "__main__":
-    main()
+    main()
```
Comment (m0nhawk) [inline pypfb/__main__.py]: Add newline in the end of file.

Comment (m0nhawk) [reply pypfb/__main__.py]: This can be changed later with autoformatting. 👍

File: README.md (hunk 2 above)
Comment (giangbui) [reply README.md]: Fixed in 395749242c00d4562e524dd8c22e3c2ea8887231

File: pypfb/__main__.py (hunk 3 above)
Comment (giangbui) [reply pypfb/__main__.py]: fixed in 395749242c00d4562e524dd8c22e3c2ea8887231

File: README.md (hunk 1 above)
Comment (giangbui) [reply README.md]: Fixed in 395749242c00d4562e524dd8c22e3c2ea8887231

**Pull request merged**